src/fileformat/types/dotnet_types/dotnet_type_reconstructor.cpp

```diff
@@ -840,6 +840,12 @@ std::unique_ptr<DotnetClass> DotnetTypeReconstructor::createClassReference(const
 	else
 	{
 		auto assemblyRefTable = static_cast<const MetadataTable<AssemblyRef>*>(metadataStream->getMetadataTable(MetadataTableType::AssemblyRef));
+
+		if (!assemblyRefTable)
```

Comment:
Would it be possible/desirable to create a regression test for `fileinfo` from of the input binaries from #458? For example from the smaller one.

Reply:
Sure, unfortunately the smaller binary contains no parsable `TypeRefTable`, therefore i made RT with the bigger one. https://github.com/avast-tl/retdec-regression-tests/pull/19